orttraining/orttraining/test/python/orttraining_test_onnxblock.py

```diff
@@ -416,10 +408,6 @@ def bcewithlogits_loss(prediction, target):
         # assert loss is close
         assert np.allclose(ort_outs[0], _to_numpy(torch_outs))
 
-        # assert all the gradients are close
```

Comment:
why delete this check?

Reply:
After changes from `InPlaceAccumulatorV2`, the gradients are no longer outputs from the model execution. And since these tests are running with the accumulation buffer as inputs from a `numpy` array on cpu, the accumulation will not be done in place leading to the inputs not being updated.
So, I cannot test for gradient closeness, unless I have access to the on device buffer. I can update the test so that we pass in inputs on GPU, but we are already testing for correctness elsewhere. This test is mostly a test for ability to execute.

Reply:
We should definitely add the test for gradient correctness in python when we make the python bindings for training Apis.